projects/cashmere/src/lib/app-switcher/app-switcher.component.scss

```diff
@@ -37,6 +37,10 @@
 
                 & > .thumbnail-img {
                     width: auto;
+
+                    &.highlight-current {
+                        opacity:0.5;
```

Comment:
I think dropping the opacity of the current app works for now.  If we get to the point of distinguishing what apps the user has access to, then we'll need to differentiate those from the current app.  But we can cross that bridge when we get to it.  @corykon - any other suggestions?

Reply:
@andrew-frueh I'd think if an app isn't available (when we get there), we may not show it in the list at all. If we do show disabled apps (perhaps cause we want people to see the apps and ask for them?), we might want opacity for those.

Opacity is probably alright for now for the current app, but I'd add a tooltip (`[title]` attribute) that says something like "current application" to clarify why its disabled just in case it's not obvious to some.

I played around with some other ideas while listening to a class lecture, but I don't know that anything is really any better.
![Artboard – 2](https://user-images.githubusercontent.com/12416432/70367282-cab26180-185b-11ea-8eed-cb2fdee2a8f0.png)

Reply:
Ugh - you have a lecture on Friday nights!  That's rough.  But thanks for the suggestions, I took two of your ideas and just merged them into the branch.  I have the opacity dropping on both the icon and the text now as per your first example.  And I also added logic to have the title say "Current Application" if it's grayed out.